django/template/utils.py

```diff
@@ -48,6 +48,12 @@ def templates(self):
                     },
                 },
             ]
+        elif settings.TEMPLATE_DIRS:
+            warnings.warn(
+                "The TEMPLATE_DIRS setting was deprecated in Django 1.8 and "
+                "the TEMPLATES dictionary takes precedence. You must put your "
+                "template directory setting into the DIRS value of your default "
```

Comment:
"template directory setting" presumably means `TEMPLATE_DIRS` here, but should probably be referenced as such, given both settings refer to 1-N template directories.